src/contract_wrappers/modules/sto/usd_tiered_sto_wrapper/3.1.0.ts

```diff
@@ -217,6 +269,46 @@ export class USDTieredSTO_3_1_0 extends USDTieredSTOBase_3_1_0 {
     return typedResult;
   };
 
+  /**
+   * Return symbol of the denominated currency
+   * @return token symbol string
+   */
+  public denominatedCurrency = async () => {
+    const result = await (await this.contract).denominatedCurrency.callAsync();
+    return bytes32ToString(result);
+  };
+
+  /**
+   * Modifies oracle
+   */
+  public modifyOracles = async (params: ModifyOraclesParams): Promise<PolyResponse> => {
+    assert.assert(
+      await this.isCallerTheSecurityTokenOwner(params.txData),
+      ErrorCode.Unauthorized,
+      'The caller must be the ST owner',
+    );
+    assert.assert(params.denominatedCurrencySymbol !== '', ErrorCode.InvalidData, 'Invalid denominatedCurrencySymbol');
+    assert.assert(
+      params.customOracleAddresses.length === 2,
+      ErrorCode.InvalidLength,
+      'Invalid customOracleAddresses length',
+    );
+    return (await this.contract).modifyOracles.sendTransactionAsync(
+      params.customOracleAddresses,
+      stringToBytes32(params.denominatedCurrencySymbol),
+      params.txData,
+      params.safetyFactor,
+    );
+  };
+
+  /**
+   * Returns the custom oracle address
+   */
+  public getCustomOracleAddress = async (params: GetCustomOracleAddressParams) => {
+    const result = await (await this.contract).getCustomOracleAddress.callAsync(params.fundRaiseType);
+    return result;
```

Comment:
Instead of returning the array directly, it would be great if it returns an object instead: 
`{
  EthOracle: string,
  PolyOracle: string,
}`
This way SDK user doesn't need to know the order of both oracles and it's explicit on the object.

Reply:
Currently getCustomOracleAddress only returns a Promise<string>

Reply:
You're right. My bad.